File: src/main/java/com/graqr/threshr/model/redsky/product/Distribution.java
package com.graqr.threshr.model.redsky.product;

import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Distribution(
        Long rating1,
        Long rating2,
        Long rating3,
        Long rating4,
        Long rating5) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/Distribution.kt
package com.graqr.threshr.circleStore.model.product

import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Distribution(
    val rating1: Long, val rating2: Long, val rating3: Long, val rating4: Long, val rating5: Long
) {}
